README.md

```diff
@@ -142,14 +146,24 @@ Model.reset_column_information
 
 # JavaScript
 * Use CoffeeScript over JavaScript
+* Namespace javascript objects for related chuncks of funcationlity (Components, Engines, etc...)  [example implementation](samples/js_namespace.md)
```

Comment:
I would really like to see a "Unit test JS objects wherever possible" added as the next point here